package/python-sip/python-sip.mk

```diff
@@ -4,9 +4,9 @@
 #
 ################################################################################
 
-PYTHON_SIP_VERSION = 4.18
+PYTHON_SIP_VERSION = 4.19.23
```

Comment:
pypi has sip at 5.3.0 - just curious what the reason for this version is?

Reply:
When I went to build the pyqt5 package it recommended this version, I worried that if I moved to a newer version it might break things.  So I opt'd for what the build wanted to be safe.  I did not want this to turn into a debug rat hole of package versioning doom.